lib/kubernetes-deploy/render_task.rb

```diff
@@ -70,7 +70,7 @@ def render_filename(filename, stream)
       rendered_content = @renderer.render_template(filename, file_content)
       implicit = true
       YAML.parse_stream(rendered_content, "<rendered> #{filename}") { |d| implicit = d.implicit }
-      stream.puts "---\n" if implicit
+      stream.puts "---\n" if implicit && rendered_content.present?
```

Comment:
I'm tempted to log when we come across an empty template. Is that overkill?

Reply:
I think that's a good idea

Reply:
Added in the last commit:

![image](https://user-images.githubusercontent.com/4789493/56514953-ab510900-6504-11e9-8eef-372b894cf4ba.png)